app/assets/javascripts/checkbox_control.es6

```diff
@@ -0,0 +1,50 @@
+$(function() {
+
+  /*
+    Usage:
+      Required: A group of checkboxes,
+        and another control checkbox outside the group
+
+      Enclose the checkbox group with a div of class "deselect-group"
+        and set it's "data-deselect-ctrl" to the jquery identifier for the control checkbox
+  */
+  $('.deselect-group').each(function() {
+    let container = $(this);
+    let control = $(container.data('deselect-ctrl'));
+    let checkboxMemory = [];
+    let checkboxes = container.find("input:checkbox");
+    let controlExists = control.length
```

Comment:
This is the only line where `;` is missing.
But `;` is actually not necessary so you can remove them completely

Reply:
Old dog and all that :). I'll add the one rather than remove the many.